### ISSUE TITLE
Week 1: The basics of ConvNets quiz question no 9

### ISSUE THREAD

**Issue by abhishekkydv8** [HeroKillerEver/coursera-deep-learning#10]
Question No 9:
In lecture, we talked about “parameter sharing” as a benefit of using convolutional networks. Which of the following statements about parameter sharing in ConvNets are true? (Check all that apply.)

   1. It allows parameters learned for one task to be shared even for a different task (transfer learning).
   2. It reduces the total number of parameters, thus reducing overfitting.
   3. It allows gradient descent to set many of the parameters to zero, thus making the connections        sparse.
   4. It allows a feature detector to be used in multiple locations throughout the whole input image/input volume.

The answer should be:

-  It reduces the total number of parameters, thus reducing overfitting.

-  It allows a feature detector to be used in multiple locations throughout the whole input image/input volume.

**Comment by tejeshvaish** [comment]
yes this is correct

**Comment by HeroKillerEver** [comment]
Thanks, I have updated it.
